config/sem_dedup_config.yaml

```diff
@@ -11,13 +11,11 @@ write_embeddings_to_disk: true
 # Clustering configuration
 clustering_save_loc: "clustering_results"
 n_clusters: 1000
-seed: 1234
 max_iter: 100
 kmeans_with_cos_dist: false
 
 # Semdedup configuration
 which_to_keep: "hard"
-largest_cluster_size_to_process: 100000
```

Comment:
IIRC, I removed `seed` and `largest_cluster_size_to_process` from the semantic dedupe parameters because I couldn't find them being used anywhere.